src/libsystemd-network/sd-dhcp-client.c

```diff
@@ -1272,6 +1272,9 @@ static int client_timeout_resend(
                 next_timeout = client_compute_request_timeout(time_now, client->attempt);
                 break;
 
+        default:
+                assert_not_reached();
```

Comment:
This needs an error message as a parameter, otherwise the build breaks

Reply:
You are right. Sorry.

Should be fixed now.